sidecred.go

```diff
@@ -57,6 +57,27 @@ func (r *Request) UnmarshalConfig(target interface{}) error {
 	return nil
 }
 
+// areConfigsEqual is a convenience function for unmarshalling the JSON config
+// from the request and resource structures, and performing a logical deep
+// equality check instead of a byte equality check. This avoids errors due to
+// structural (but non-logical) changes due to (de)serialization.
+func areConfigsEqual(b1, b2 []byte) bool {
```

Comment:
Nitpick; perhaps `isEqualConfig` would be better here since it rolls a bit easier off the tongue around if statements? Is sounds weird since it takes two objects... but then again we have `assert.IsEqual` (and not `assert.AreEqual`)🤷‍♂ What do you think?

Reply:
Yes, I agree. It sounds much better.